app/controllers/krikri/validation_reports_controller.rb

```diff
@@ -19,14 +19,19 @@ class ValidationReportsController < CatalogController
 
     def show
```

Comment:
Can we add a comment here about how this diverges from how `Blacklight::CatalogController` expects `#index` and `#show` to be used?